script.js

```diff
@@ -0,0 +1,124 @@
+var nouns = [];
+var adjectives = [];
+var adverbs = [];
+var verbs = [];
+console.log('nouns1:',nouns);
+var story;
+
+var backupNouns = ["parent", "car", "zebra", "incense"];
+var backupAdjectives = ["blue", "lovely", "grainy", "black"];
+var backupAdverbs = ["gracefully", "foolishly", "often", "obnoxiously"];
+var backupVerbs = ["burp", "walk", "swing", "scream"];
+
+function generateMadLib() {
+	nouns = [];
+	adjectives = [];
+	adverbs = [];
+	verbs = [];
+	var addNouns = function() {
+		var userNouns = document.getElementById('nouns').value;
+		userNouns = userNouns.split(",");
+		nouns = nouns.concat(userNouns);
+		nouns.shift();
+		console.log('nouns2:',nouns);
```

Comment:
Typically it is "best practice" to remove your console.logs before your code is ready for a "production" environment. Although this class is not necessarily "production" but it is best to get into good habits now.